### PR TITLE
update ruby/bundler version

### PR DESCRIPTION
**Pull request by franklin-stripe**
* Update ruby versions to > 2.4
* Update bundler to 2.0

I was having some problems with my other PR, and it looks like it's because bundler isn't supported on older version fo Ruby anymore. Bumping these to be more recent should fix the issue

### CODE REVIEW

File: qbo_api.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.10"
+  spec.add_development_dependency "bundler", "~> 2.0"
```
Comment (minimul) [inline qbo_api.gemspec]: Instead let's go with ">= 1.10"

Comment (franklin-stripe) [reply qbo_api.gemspec]: fixed!

**Comment by minimul** [comment]
I'm going merge in #104 and release it in the 1.8.x series and then this PR will be merged and released as 1.9.0.

**Comment by minimul** [comment]
@franklin-stripe Please squash into a single commit. Thanks!

**Comment by franklin-stripe** [comment]
@minimul done! 

FWIW I think Github has a "merge + squash" setting you can enable to do it straight from the UI

**Comment by minimul** [comment]
Another thanks @franklin-stripe – released as 1.9.0.

**Pull request merged**